ctrd/container_types.go

```diff
@@ -8,11 +8,13 @@ import (
 
 // Container wraps container's info.
 type Container struct {
-	ID      string
-	Image   string
-	Runtime string
-	IO      *containerio.IO
-	Spec    *specs.Spec
+	ID       string
+	Image    string
+	Runtime  string
+	IO       *containerio.IO
+	Spec     *specs.Spec
+	BaseFS   string
+	Takeover bool
```

Comment:
More description for the whole Container struct. In addition, I wish that we could use `CtrdContainer` instead of `Container`, since we have already so many Container struct in pouchd.

Currently, with the change, this struct will handle two kinds of container, the first one is the original one, and the second is the ones from Moby and taken over by Pouch. This info is quite important to be added in the comments of this struct.

Also, every field needs comments.

Reply:
`CtrdContainer` will cause glint error: it recommands to use  name `Container` :)

i will add more details description.